keyboards/1upkeyboards/1upslider8/info.json

```diff
@@ -0,0 +1,116 @@
+{
+    "manufacturer": "1upkeyboards",
+    "keyboard_name": "1upslider8",
+    "maintainer": "ziptyze",
+    "processor": "RP2040",
+    "bootloader": "rp2040",
+    "board": "GENERIC_RP_RP2040",
```

Comment:
I don't believe that this is actually needed.  It should be inferred based on the processor.
```suggestion
```